geopy/distance.py

```diff
@@ -122,12 +123,17 @@
 
 def lonlat(x, y, z=0):
     """
-    Helper function that can be used for more explicit and obvious calling of the distance methods
+    distance accepts coordinates in (y, x)/(lat, lon) order, while some other libraries/systems might use 
```

Comment:
1. A sentence should start with a capital letter :). But in this case `distance` is a term (a class reference, more specifically), not a plain word. See the docs around for examples on how `distance` is formatted there.
2. Please try to keep the line lengths close to 80-ish.
3. Coordinate pairs should also be formatted as terms (with backticks, I guess), otherwise they might be split by word-wrapping.